vsintegration/src/FSharp.Editor/LanguageService/FSharpCheckerProvider.fs

```diff
@@ -57,12 +57,12 @@ type internal FSharpCheckerProvider
             let checker = 
                 FSharpChecker.Create(
                     projectCacheSize = settings.LanguageServicePerformance.ProjectCheckCacheSize, 
-                    keepAllBackgroundResolutions = false,
+                    keepAllBackgroundResolutions = true,
```

Comment:
This is temporarily 'true'. It will be 'false' once we figure out a way to enable background resolutions only for open documents.